_posts/2021-02-19-26-rln-relay.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+name:  "Integrating rate limiting nullifiers into Waku Relay to enable spam-protection."
+title:  "Integrating rate limiting nullifiers into Waku Relay to enable spam-protection."
+date:   2021-02-25 12:00:00 +0800
+author: staheri14
+published: true
+permalink: /waku-v2-rln-relay
+categories: research
+summary: A simplified explanation of how Waku RLN-Relay mitigates the spamming issue in a p2p manner.
+image: 
```

Comment:
Picture

Reply:
Where does this picture show up on the website? I wanna get a feeing of what sort of picture is needed

Reply:
Discussed